moto/wafv2/models.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import unicode_literals
+
+import uuid
+
+from moto.core import BaseBackend, BaseModel
+from moto.wafv2.utils import make_webacl_arn
+
+US_EAST_1_REGION = "us-east-1"
+GLOBAL_REGION = "global"
+
+
+class WAFV2Backend(BaseBackend):
+    """
+    https://docs.aws.amazon.com/waf/latest/APIReference/API_Types_AWS_WAFV2.html
+    """
+
+    def __init__(self, region_name=None):
+        super(WAFV2Backend, self).__init__()
+        self.region_name = region_name
+        self.wacls = [WebACL("first-mock-webacl"), WebACL("second-mock-webacl")]
+
+    def reset(self):
+        region_name = self.region_name
+        self.__dict__ = {}
+        self.__init__(region_name)
+
+
+class WebACL(BaseModel):
+    def __init__(self, name="Mock-WebACL-name"):
+        self.Id = str(uuid.uuid4())
+        """
+        The name must have 1-128 characters. Valid characters: A-Z, a-z, 0-9, - (hyphen), and _ (underscore).
+        """
+        self.Name = "{0}-{1}".format(name, uuid.uuid4())[0:128]
+        self.ARN = make_webacl_arn(US_EAST_1_REGION, self.Name)
+        self.Capacity = 3
+        self.Description = "Mock WebACL named {0}".format(self.Name)
+        self.VisibilityConfig = VisibilityConfig(self.Name)
+        """
+        TODO: create classes for the below fields:
```

Comment:
There are many [other fields in a WebACL](https://docs.aws.amazon.com/waf/latest/APIReference/API_WebACL.html), and those, in turn, often have many other fields, too. Should I put them all in this `models.py` file, or should I create a `models` package with the other classes?

Reply:
It's fine to have them all in the same file.
For most services, if there's a model with many (nested) fields, we tend to just store them as a dictionary. Depending on how many of the fields are actually used, or whether they're simply stored for retrieval later

Reply:
Attribute-names should be pythonic btw (snake_case, so `self.name` and `self.visibility_config`)